### ISSUE TITLE
Logging and locations of basic-auth password failures

### ISSUE THREAD

**Issue by McFly99** [Screenly/screenly-ose#620]
Greetings,

I can't seem to find logging of a failure of the basic-auth credentials.

Am I mistaken on this?

It would be really nice to be able to have a failure result in a log file somewhere as then a person can tie fail2ban to it. I looked for it in the nginx logs to no avail using ssl on a .10 image updated and setup with basic-auth.

**Comment by vpetersson** [comment]
Yeah, i don't think we log anything like that. If you want that level of security, I'd probably put Nginx in front of it with basic auth, and then reverse proxy to Screenly. Just make sure you change the binding to localhost and you should be good.
